Bylaws.mediawiki

```diff
@@ -70,6 +70,9 @@
 ## ensuring that they are subscribed to the members-announce email list.
 ## ensuring they have signed a copy of the current version of the Hacklab waiver.
 ## paying dues as set by the board.
+### A member will be dememberized if they are at least three months in arears on their dues and have received at least two notices from the board about their arears.
```

Comment:
"membership revoked" instead of "dememberized" for consistency with the rest of the text
"notices from the board or treasurer" since it's mostly the treasurer sending out the notices
"arrears" not "arears"